platform/linux-generic/include/odp/api/plat/ipsec_inlines.h

```diff
@@ -36,6 +41,23 @@ _ODP_INLINE odp_event_t odp_ipsec_packet_to_event(odp_packet_t pkt)
 	return odp_packet_to_event(pkt);
 }
 
+_ODP_INLINE int odp_ipsec_result(odp_ipsec_packet_result_t *result, odp_packet_t pkt)
+{
+	odp_ipsec_packet_result_t *res;
+
+	_ODP_ASSERT(result != NULL);
+	_ODP_ASSERT(odp_packet_subtype(pkt) == ODP_EVENT_PACKET_IPSEC);
+
+	res = _odp_pkt_get_ptr(pkt, odp_ipsec_packet_result_t, ipsec_ctx);
+
+	/* FIXME: maybe postprocess here, setting alg error in case of crypto
+	 * error instead of processing packet fully in ipsec_in/out_single */
```

Comment:
This comment does not appear very useful so it could be removed.

Reply:
Removed in V2.